desktop-src/AD/security-properties.md

```diff
@@ -232,6 +232,17 @@ The [**scriptPath**](/windows/desktop/ADSchema/a-scriptpath) attribute specifies
 
 </dd> <dt>
 
+<span id="tokenGroups"></span><span id="tokengroups"></span><span id="TOKENGROUPS"></span>[**tokenGroups**](/windows/win32/adschema/a-tokengroups)
```

Comment:
This looks good, but I have a minor suggestion. The "/windows/desktop" path now redirects to "/windows/win32". For any new or updated content, we should use the new path to avoid redirects. For tokenGroups, it would be /windows/win32/adschema/a-tokengroups
Thanks!